src/frontend/components/UI/Sidebar/components/SidebarLinks/index.tsx

```diff
@@ -234,45 +181,20 @@ export default function SidebarLinks() {
                 ['active']: type === 'games_settings'
               })}
             >
-              {isDefaultSetting && (
-                <span>
-                  {t(
-                    'settings.navbar.games_settings_defaults',
-                    'Game Defaults'
-                  )}
-                </span>
-              )}
-              {!isDefaultSetting && (
-                <span>
-                  {t('settings.navbar.games_settings', 'Game Settings')}
-                </span>
-              )}
+              <span>
+                {t('settings.navbar.games_settings_defaults', 'Game Defaults')}
+              </span>
+            </NavLink>
+            <NavLink
+              role="link"
+              to={`/settings/${runner}/${appName}/advanced`}
+              state={{ ...state, runner: state?.runner }}
+              className={classNames('Sidebar__item SidebarLinks__subItem', {
+                ['active']: type === 'advanced'
+              })}
+            >
+              <span>{t('settings.navbar.advanced', 'Advanced')}</span>
             </NavLink>
-            {cloud_save_enabled && !isLinuxGame && (
-              <NavLink
-                role="link"
-                data-testid="linkSync"
-                to={`/settings/${runner}/${appName}/sync`}
-                state={{ ...state, runner: state?.runner }}
-                className={classNames('Sidebar__item SidebarLinks__subItem', {
-                  ['active']: type === 'sync'
-                })}
-              >
-                <span>{t('settings.navbar.sync')}</span>
```

Comment:
I think we are losing access to the sync saves config now? I can' t test this at the moment to check

Reply:
That's totally true. I can't believe I missed that. Thanks for finding this.

Yes, I will need to render the save sync section here as well. I will move it to game settings.